### ISSUE TITLE
"Outdated Trezor firmware" error blocks setup

### ISSUE THREAD

**Issue by jlopp** [spesmilo/electrum#5391]
I was just testing our open source "sovereign recovery" process for Casa 3-of-5 multisig users and noticed that it no longer works for me due to the "outdated firmware" error.

Because Casa's multisig key management app is seedless (users don't store the seed phrases for devices) it's likely that their device firmware may be out of date. This is because firmware updates sometimes wipe the seed from the device, thus requiring the user to perform a key rotation. Also, hardware devices are geographically distributed and may not be used for long periods of time.

Long story short, if a catastrophic event were to befall Casa and our users need to recover funds without using our servers to coordinate signing, they'll need to be able to recreate their wallets using their hardware devices with whatever the currently running firmware version is, and upgrading the firmware may not be an option. My proposal is that this blocker error be changed to a non-blocking warning, or that some alternative method be made available that can bypass this block.

**Comment by jlopp** [comment]
It appears that this is more of an issue with the Trezor library as I'm experiencing the same issue when using Bitcoin Core's HWI. From what I can tell, the version check is arbitrary and the functionality I expect still works if I comment out the version checking.

**Comment by jlopp** [comment]
https://github.com/trezor/python-trezor/issues/378

**Comment by SomberNight** [comment]
see discussion in https://github.com/spesmilo/electrum/pull/4875#issuecomment-442133479

> The point of the new firmware checks is to enforce that people don't use old FWs with security problems, that's why I suggested auto-checking. python-trezor itself will not do silent pings. But I'll bundle latest-known-good FW images and check if yours is older than that.
As I intend to do it now, too-old firmware will raise exceptions by default. I'm thinking of making it possible to ignore them but having noisy warning boxes in the UI?

**Comment by SomberNight** [comment]
> I'm thinking of making it possible to ignore them but having noisy warning boxes in the UI?

^ done

**Comment by jlopp** [comment]
Thanks!
